css-dev/burf-base.scss

```
@@ -4,7 +4,8 @@
 
 // dependencies and resets
 @import "normalize/import-now";
-@import "math";
+@import "node_modules/normalize-scss/sass/normalize/import-now";
```

I think we want to use the tilde`~` style imports here for consistency. 

```suggestion
@import "~/normalize-scss/sass/normalize/import-now";
```